tests/e2e/docker/Dockerfile

```diff
@@ -71,6 +71,8 @@ RUN curl -fsSL https://download.docker.com/linux/debian/gpg | gpg --dearmor -o /
 RUN curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose && \
   chmod +x /usr/local/bin/docker-compose
 
+RUN grep -q mozilla/DST_Root_CA_X3.crt  "/etc/ca-certificates.conf" && sed -e '/mozilla\/DST_Root_CA_X3.crt/s/^/!/g' -i /etc/ca-certificates.conf && update-ca-certificates
```

Comment:
please use long option names in Dockerfiles (i.e. instead of ```-i``` use ```--in-place```

Reply:
MINOR: use multiline, i.e. with ``\``, to visualize easier changes and options

Reply:
@Surfict ??